kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/tokenstore/KafkaTokenStore.java

```diff
@@ -0,0 +1,615 @@
+         * min.compaction.lag.ms} of the topic should be at least double the {@code claimTimeout}. This is because we
+         * consider the first message with the same sequence number as the 'correct' message, instead of Kafka, that
+         * might delete a message when there is a newer message using the same key. To be sure the topic is cleaned up
+         * in time, independent of the default configured values, the {@code max.compaction.lag.ms} can be set to a
+         * value about 4 times the {@code min.compaction.lag.ms} value.
+         *
+         * @param topic the topic used for token claim updates
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public Builder topic(String topic) {
+            assertNonEmpty(topic, "The topic may not be null");
+            this.topic = topic;
+            return this;
+        }
+
+        /**
+         * Sets the {@link Serializer} used to de-/serialize {@link TrackingToken}s with.
+         *
+         * @param serializer a {@link Serializer} used to de-/serialize {@link TrackingToken}s with
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public Builder serializer(Serializer serializer) {
+            assertNonNull(serializer, "Serializer may not be null");
+            this.serializer = serializer;
+            return this;
+        }
+
+        /**
+         * Sets the {@code claimTimeout} specifying the amount of time this process will wait after which this process
+         * will force a claim of a {@link TrackingToken}. Thus, if a claim has not been updated for the given {@code
+         * claimTimeout}, this process will 'steal' the claim. Defaults to a duration of 10 seconds.
+         *
+         * @param claimTimeout a timeout specifying the time after which this process will force a claim
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public Builder claimTimeout(TemporalAmount claimTimeout) {
+            assertNonNull(claimTimeout, "The claim timeout may not be null");
+            this.claimTimeout = claimTimeout;
+            return this;
+        }
+
+        /**
+         * Sets the {@code nodeId} to identify ownership of the tokens. Defaults to the name of the managed bean for the
+         * runtime system of the Java virtual machine.
+         *
+         * @param nodeId the id as a {@link String} to identify ownership of the tokens
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public Builder nodeId(String nodeId) {
+            assertNonEmpty(nodeId, "The nodeId may not be null or empty");
+            this.nodeId = nodeId;
+            return this;
+        }
+
+        /**
+         * Sets the {@code executor} to identify ownership of the tokens. Defaults to the name of the managed bean for
```

Comment:
The default description is off here. Added, I'd expect this default to be described in the `builder()` and `class Builder` JavaDocs too.

Reply:
Fixed